js/scripts.js

```diff
@@ -1,3 +1,17 @@
 function getTriangleArea(a, h) {
-	
-}
\ No newline at end of file
+	if (a > 0 && h > 0) {
+		return a*h/2;
+	} else if (a <= 0 || h <= 0) {
```

Comment:
`else if` is unnecessary, you can write simple return in this case, because in earlier code you have